packages/perseus/src/widgets/__stories__/test-keypad-context-wrapper.tsx

```diff
@@ -12,6 +13,11 @@ const Footer = (): React.ReactElement => {
                         onElementMounted={setKeypadElement}
                         onDismiss={() => renderer && renderer.blur()}
                         style={styles.keypad}
+                        useV2Keypad={true}
```

Comment:
@handeyeco I hope/think this is ok. I think at this point we want to be testing with the new keypad most places. Let me know what you think.

Reply:
Said PR: https://github.com/Khan/perseus/pull/705